src/cheri_sys_regs.sail

```diff
@@ -65,6 +65,8 @@
 /* Capability control csr */
 
 bitfield ccsr : xlenbits = {
+  /* Bits allocated from 31 downwards are used for feature flags. */
+  tc      :  31,     /* tag-clearing error semantics */
```

Comment:
I am planning to add bit 30 for "DDC relocation disabled" here as I don't believe we want that to be writable as it is for Morello.

Reply:
That seems fine to me.